### ISSUE TITLE
Particl desktop initialization hangs on linux

### ISSUE THREAD

**Issue by aclifton314** [particl/particl-desktop#1857]
# Describe bug/issue
Initial launch of Particl desktop appears to hang (~3hours with no progress).

# How to reproduce
1. Download linux deb file `particl-desktop-3.2.0-linux-amd64.deb`
2. Install via `sudo apt install /path/to/particl-desktop-3.2.0-linux-amd64.deb`
3. Launch  particl desktop from command line using `Particl\ Desktop`

# More information
- Operating system and version: Pop!_OS 22.04 (basically Ubuntu 22.04)
- Client version: 3.2.0

running the cli command produces a GUI with nothing on it and also displays the following in the terminal:
```
aclifton@pop-os:~$ Particl\ Desktop
port=51735
[17:10:37.453] [info] init particl-desktop : 3.2.0
[17:10:37.548] [info] initialization complete
[17:10:37.549] [info] Start system called
[17:10:37.550] [info] Building up UI elements...
```
It hangs at `Building up UI elements...`. I left it alone here for 3 hours and didn't see it progress. I discovered particl today and am super excited to test it out and explore it. I'm sure there is something silly I'm doing on my end and any feedback is greatly appreciated. I tried to uninstall and reinstall but couldn't figure out the most complete and appropriate way to do so. 

Thanks in advance for your help!

**Comment by zaSmilingIdiot** [comment]
There is currently an issue with the ability for the Particl Desktop GUI to correctly start up on various "newer" updates/releases of Linux distributions. This has been fixed upstream in electron, and we have been recently migrating from an older electron version to a more recent release (which should also help with various Wayland related issues if those are being experienced as well).

The change will be available in the v3.2.1 release, which was just made available for testing purposes, ie: on testnet. See the Releases section of this repo for the build if you'd like to try it out. The mainnet builds for v3.2.1 should then be available in a couple of days.

**Comment by aclifton314** [comment]
@zaSmilingIdiot thank you very much for your response and clear explanation. I will go ahead and close this issue and wait for v3.2.1. If I happen to find the same issue in that version, I'll reopen the issue. If you prefer to keep it open, please feel free to do so.
